ui/app/mainui/Popups.qml

```diff
@@ -739,9 +742,21 @@ QtObject {
                 Connections {
                     target: root.rootStore.communitiesModuleInst
 
-                    function onSharedAddressesForAllNonKeycardKeypairsSigned() {
+                    function onAllSharedAddressesSigned() {
```

Comment:
Just comment for the future refactor: Wondering if we can "move" or implement this slot somehow differently bc I've seen this code like in 3 or 4 different qml files the same way. Big risk of introducing bugs here!